src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -134,10 +143,26 @@ public User getUser(String email) {
     if (userEntity == null) {
       return null;
     }
-
     // Return aboutMe
     String aboutMe = (String) userEntity.getProperty("aboutMe");
     User user = new User((String) userEntity.getProperty("email"), aboutMe);
     return user;
   }
+  /** Returns true if country is stored in datastore, false o.w. */
+  public Country getCountry(String countryCode) {
```

Comment:
If a country exists, should this return true (as in the comment), or should it return the country object?

Reply:
Good catch, I originally had it as a isCountryValid message and forgot to change the comment. I'll do that :D